backend/requirements.txt

```diff
@@ -0,0 +1 @@
+flask-alchemy
```

Comment:
Here we're missing a few. You can use this for example to trace them https://github.com/jazzband/pip-tools